src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,38 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on your Project Board] to learn more on how to work with
+projects.
 
 ### Backlog Board
 
-To be expanded
+[The backlog board] hosts issues or features that are intended to be worked upon. While each issue is opened in
+its corresponding repository, this board gives an aggregated view of all open issues across all repositories. The board has total of seven
+columns. Any issue or epic opened should be assigned to `Backlog` Projects in `Backlog` column. Issues intenteded to be
+worked on for the next release are moved to `Next Version Candidate` column. During [pre-planning](#Pre-Planning), issues from the `Next
+Version Candidate` column are triaged. Issues or epics properly triaged, with assignees and priority labels addeded, are moved to [current
+release board](#Current Release Board). The next 3 columns are of priority. All unassigned issues are sorted into these three columns
+based on the priority discussed in the team meeting. Issues from these columns should be worked upon if the current release work is
+finished. Any leftover work from the previous release goes to the `Work in Progess column`. `Close?` column holds any issue/epic that
+could probably be closed for various reasons.
 
 ### Current Release Board
 
-To be expanded
+Current release work is tracked in [the 0.13 board]. THe board has 5 columns. The first one, `Schedule and Epics`, hosts all properly
+triaged epics targeted for the current release and schedule for the current release. Triaged issues are under `TO DO` column. When an
```

Comment:
```suggestion
triaged epics targeted for the current release and schedule for the current release. Triaged issues are under the `TODO` column. When an
```

Reply:
The column is named "To do" with the space. I changed the case.